example/nomad_acl_test.tf

```diff
@@ -0,0 +1,11 @@
+data "vault_generic_secret" "nomad_secret_id" {
+  # Set count of this data source to 1 if ACLs are enabled in Nomad, and 0 if not
+  count = var.nomad_acl ? 1 : 0
+  path  = "nomad/creds/write"
+}
+
+provider "nomad" {
+  address = "http://127.0.0.1:4646"
+  # Add a secret_id if ACLs are enabled in nomad
+  secret_id = var.nomad_acl ? data.vault_generic_secret.nomad_secret_id[0].data.secret_id : null
+}
```

Comment:
What is this for?

Reply:
Give token to provider, once ACL enabled. A test-matrix is running against the module.

https://github.com/fredrikhgrelland/terraform-nomad-hive/runs/1031289801?check_suite_focus=true#step:8:424

Reply:
Now all tests are green :)